Microsoft.Toolkit.Uwp.SampleApp/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.1.0",
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
```

Comment:
that's why i only updated the app.. when i updated it across board, it couldn't find OrderedDictionary. having 5.2.2 on sample app does not impact the libraries in any ways